doc/rtmpwebhookauth.md

```diff
@@ -1,24 +1,28 @@
 # RTMP Webhook Authentication
 
-Incoming RTMP streams can be authenticated using Webhooks. To use it you need to implement own web server which will be called by Livepeer node when new RTMP stream received to check if that stream is allowed.
+Incoming RTMP streams can be authenticated using Webhooks. To provide these webhooks, node operators must implement their own web service / endpoint to be accessed only by the Livepeer node. As new RTMP streams appear, the Livepeer node will hit this endpoint to determine whether the given stream is allowed.
 
-To enable Webhook authentication functionality Livepeer node should be started with flag `-authWebhookUrl`, like this:
+To enable webhook authentication functionality, the Livepeer node should be started with the flag, `-authWebhookUrl` along with the webhook endpoint URL.
 
-`livepeer -authWebhookUrl http://ownserver/auth`
+For example:
 
-For every incoming RTMP stream Livepeer node will make `POST` request to `http://ownserver/auth` endpoint passing url to which RTMP request was made as JSON object.
+```console
+livepeer -authWebhookUrl http://ownserver/auth
+```
+
+For each incoming RTMP stream, the Livepeer node will make a `POST` request to the `http://ownserver/auth` endpoint, passing the URL of the RTMP request as JSON object.
 
-For example, if incoming RTMP request was made to `rtmp://livepeer.node:1935/something?manifestID=manifest`, Liverpeer node will pass this object in request to webhook:
+For example, if the incoming RTMP request was made to `rtmp://livepeer.node:1935/something?manifestID=manifest`, the Liverpeer node will provide the following object as a request to the webhook endpoint:
 
 ```json
 {
     "url": "rtmp://livepeer.node:1935/manifest"
 }
 ```
 
-Webhook server should respond with code 200 if it wants to allow RTMP stream. Response with any other code will lead to Livepeer node dropping RTMP stream.
+The webhook server should respond with HTTP status code `200` in order to authenticate / allow the RTMP stream. A response with a HTTP status code other than `200` will cause the Livepeer node to drop the RTMP stream.
 
-Webhook can respond with zero body - in that case `ManifestID` for the stream will be taken from url or generated randomly in case manifest id is not specified in url. Or webhook server can respond with JSON object in this format:
+The webhook may respond with an empty body.  In this case, the `manifestID` property of the stream will taken from the URL.  If the URL does not specify a manifest id, then it will be generated at random.  Otherwise, the webhook endpoint should respond with a JSON object in the following format:
```

Comment:
> will taken from the URL

"will be taken from the RTMP URL" (added 'be' and specified RTMP)

>  If the URL does not specify a manifest id

"If the RTMP URL does not specify a manifest id"

And maybe give the RTMP URLs that are considered to be missing manifest IDs: `rtmp://host` and `rtmp://host/stream`

Reply:
partially addressed in 2d7244324bdade66d0e440cc46273801e52d29bf